android/src/main/java/sqip/flutter/internal/CardEntryModule.java

```diff
@@ -1,29 +1,24 @@
-/*
```

Comment:
We should either have the license header or not, throughout?  If it's inconsistent today, don't change that in this PR, though.